### PR TITLE
CHANGELOG: add v0.2.0

### PR DESCRIPTION
**Pull request by iaguis**
This is a draft PR to review the changelog for v0.2.0.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,210 @@
...
+#### Prerequisites
+
+Some component objects changed `apiVersion` so they need to be labeled and annotated manually to be able to upgrade them.
+
+##### Dex
+
+```
+kubectl -n dex label ingress dex app.kubernetes.io/managed-by=Helm 
+kubectl -n dex annotate ingress dex meta.helm.sh/release-name=dex
+kubectl -n dex annotate ingress dex meta.helm.sh/release-namespace=dex
```
[the remark this answers is not in the document]
Comment (iaguis) [reply CHANGELOG.md:29]: I think it's more clear this way so users can see what changed in one quick glance.

**Comment by iaguis** [comment]
Addressed comments and added a line to install the new Calico CRD that I forgot before.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,266 @@
...
+    }
+}
+```
+
+New:
+
+```hcl
+dns {
+    zone = "<DNS_ZONE>"
+    provider = "route53"
```
Comment (invidian) [inline CHANGELOG.md:93]:
```suggestion
    zone     = "<DNS_ZONE>"
    provider = "route53"
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,266 @@
...
+
+Here's an example of the changed syntax.
+
+Old:
+
+```hcl
+component "prometheus-operator" {
+    namespace = "<NAMESPACE>"
+    grafana_admin_password = "<GRAFANA_PASSWORD>"
+    etcd_endpoints = ["<ETCD_IP>"]
```
Comment (invidian) [inline CHANGELOG.md:115]:
```suggestion
    namespace              = "<NAMESPACE>"
    grafana_admin_password = "<GRAFANA_PASSWORD>"
    etcd_endpoints         = ["<ETCD_IP>"]
```

**Comment by ipochi** [comment]
Changing CHANGELOG.md should not trigger builds.
https://github.com/kinvolk/lokomotive-ci/pull/136

**Comment by invidian** [comment]
Done in #645

**Pull request closed without merging**